code/mob/living/silicon/hivebot/hive_modules.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/hive_module
 	name = "hive robot module"
-	icon = 'icons/obj/module.dmi'
+	icon = 'icons/obj/machinery/module.dmi'
```

Comment:
All of the `module.dmi` stuff look like circuitboard type things (from a cursory glance). Doesn't seem like machinery to me.